src/gui/sign.sh

```diff
@@ -1,7 +1,9 @@
 gui_sign() {
-    yad --file
-    #yad --title="EasyGnuPG | Sign a File" \
-    #   --text="Sign a File"
+    local file output
+    file=$(yad --file --title="EasyGnuPG | Sign a File")
```

Comment:
We should remove the `$file` in case it exists, or ask the user first whether it should be replaced.